src/components/InfoWindow.js

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from 'react';
+import { getDistanceFromLatLonInKm } from '../helpers/Haversine';
+import { LocationsContext } from '../context/LocationsContext';
+import './InfoWindow.css';
+
+export default function InfoWindow({ $dimensionKey }) {
```

Comment:
What is `$dimensionKey`? The name doesn’t immediately tell me what kind of data it might hold or even exactly its purpose. And starting it with a `$` is kind of non-standard. Was that how it was done in an example you followed? If so, I wonder why? Consider renaming it to something a bit more descriptive or at a minimum adding a comment that explains what kinds of values it will hold and what they represent. Thanks!

Reply:
We're trying to refrence the unique key we gave lcoation pins that we created via .map but react changes that prop to instead be the $dimensionKey prop in dev tools. I added a comment about it above to explain